### PR TITLE
updates school/session-type visualizations

### PR DESCRIPTION
**Pull request by stopfstedt**
this is a complete rewrite and displays a different output.

adds breadcrumb, fixes formatting while at it.

fixes https://github.com/ilios/ilios/issues/3391
fixes https://github.com/ilios/ilios/issues/4617

### CODE REVIEW

**Review by dartajax — CHANGES_REQUESTED** [review]
stale review - wrong stuff - I have since approved this

**Comment by dartajax** [comment]
The mouse was over the 8.0% slice in my first screen shot - Airways, etc is the course.

**Review by dartajax — APPROVED** [review]
Pretty slick.

**Comment by dartajax** [comment]
Ahh but when you scroll down, mouse still over the 8.0% you can see the whole thing ...

image

File: translations/fr.yaml [hunk 1]
```diff
@@ -442,9 +443,11 @@ general:
   validUsers: "utilisateurs valides"
   verificationPreviewFor: "Aperçu de vérification pour {name}"
   viewAll: "Voir Tous"
+  vocabulariesBySessionType: "Vocabulaires par Type de Seance"
   weak: Faible
   whichIs: "qui est"
   write: "Écriture"
+  xTermsFromVocabYusedWithSessionTypeZ: "{termsCount, plural, =1 {1 term} other {# termes}} du vocabulaire \"{vocabulary}\" {termsCount, plural, =1 {est appliqué} other {sont appliqués}} à {sessionsCount, plural, =1 {1 session} other {# sessions}} avec le type de session \"{sessionType}\"."
```
Comment (jrjohnson) [inline translations/fr.yaml:450]: Now that's what I call a name!

**Review by jrjohnson — APPROVED** [review]

File: translations/fr.yaml (hunk 1 above)
Comment (dartajax) [reply translations/fr.yaml:450]: Do we need some Spanish trans here?

Comment (stopfstedt) [reply translations/fr.yaml:450]: Spanish translations - courtesy of G00gle - are here: https://github.com/ilios/frontend/pull/7051/files#diff-647ea58af4907f7545a8d9536a12c7f695e0b0513eccdf84fcf17f511729f76dR449

and here 

https://github.com/ilios/frontend/pull/7051/files#diff-647ea58af4907f7545a8d9536a12c7f695e0b0513eccdf84fcf17f511729f76dR402

**Pull request merged**